semgrep/semgrep/target_manager.py

```diff
@@ -237,7 +239,7 @@ def filtered_files(self, lang: Language) -> Set[Path]:
 
         targets = self.expand_targets(directories, lang, self.respect_git_ignore)
         targets = self.filter_includes(targets, self.includes)
-        targets = self.filter_excludes(targets, self.excludes)
+        targets = self.filter_excludes(targets, self.excludes + [".git"])
```

Comment:
Should we constantize this list of default excludes in a global variable?

Reply:
Can probably wait til we add more thing